xt/00-check-changelog.t

```diff
@@ -0,0 +1,17 @@
+use Test::More tests => 1;
+
+use strict;
+use warnings;
+
+use CPAN::Changes;
+use Data::Dumper;
+
+my $changes = CPAN::Changes->load('Changes');
+
+ok($changes);
+
+my @releases = map { +{ $_->version => $_->date } } $changes->releases;
+
+note Dumper \@releases;
```

Comment:
It should still show up since I'm using `-v`, but turns out, it isn't running at all?

/me investigates.